netsec_fall2017/lab2/src/lab2_protocol/PEEPClientProtocol.py

```diff
@@ -131,10 +137,11 @@ def data_received(self, data):
 							print()
 						self.peeptransport = PEEPTransport(self.transport)
 						self.peeptransport.logging = self.logging
+						self.peeptransport.sequenceNumber = self.sequenceNumber
 						self.higherProtocol().connection_made(self.peeptransport)
 
 				elif packet.Type == 2 and self.state == "Transmission_State_2": # receiving ACK back
-					self.peeptransport.ack_received(packet.Acknowledgement,self.logging)
```

Comment:
I think you need to pass self.logging here since we want to make PEEPProtocol and PEEPTransport share a same logging switch